Rebuild-Black-Business/RBB-Website
# Fature/Allies gate

Add a gate to the ally page to make it harder to scrape and force people to agree with the Terms and Conditions

Fixes #85 

## Pages/Interfaces that will change

/allies

### Screenshots / video of changes

<img width="1650" alt="Rebuild_Black_Business_-_Rebuild_Black_Business" src="https://user-images.githubusercontent.com/209089/84197643-e47d6d00-aa67-11ea-94e5-31b84d535dce.png">

## Steps to test

1. Go to `/allies`
2. Click on the button `I accept the Terms and Conditions`
3. See the allies

### Additional notes

We store your state into local storage as a boolean at `acceptedTAC`. If you want to try it again, clear your local storage or change the value to false.
